elements/lisk-p2p/test/unit/peer_selection.ts

```diff
@@ -25,6 +25,8 @@ describe('peer selector', () => {
 		let peerList = initializePeerInfoList();
 		const nodeInfo: P2PNodeInfo = {
 			height: 545777,
+			lastBlockId: '1234',
```

Comment:
we are not adding any changes to p2p library, but we need to update the test?
I think p2p doesn't know anything about those values.

Reply:
Agreed, this was before I removed the properties from P2P as @ishantiw told me it is made to support generic ones atm.